guides/spin/cli.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: single
+title:  "Install and Configure Spin CLI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Install `spin`
+
+One way to manage applications and pipelines as code is through [spin](http://github.com/spinnaker/spin).
+
+To acquire `spin`, do the following:
+
+```bash
+export VERSION=<desired spin version tag> # Select a tag from the released version, e.g. v0.1.0
+curl -L https://github.com/spinnaker/spin/releases/download/$VERSION/spin -o spin
+chmod +x ./spin
+./spin --help
+```
+
+**Note**: The alpha `spin` binaries are only available for Linux platforms only at this point.
+We'll compile binaries for other platforms.
+
+## Configure `spin`
+
+`spin` reads its configuration from `~/.spin/config`. At this point in time, most of configuration
```

Comment:
s/most of configuration/most of its configuration

Reply:
Is there configuration not for auth?

Reply:
No good point.